forecasting/src/autogluon/forecasting/task/forecasting/forecasting.py

```diff
@@ -0,0 +1,153 @@
+from autogluon.core.task.base.base_task import BaseTask, schedulers, compile_scheduler_options
+from .dataset import TimeSeriesDataset
+from autogluon.core.dataset import TabularDataset
+from ...utils.dataset_utils import rebuild_tabular, train_test_split_dataframe, train_test_split_gluonts
+from ...learner import DefaultLearner as Learner
+from .predictor_legacy import ForecastingPredictorV1
+from ...trainer import AutoTrainer
+import pandas as pd
+from gluonts.dataset.common import Dataset, ListDataset, FileDataset
+from autogluon.core.utils.utils import setup_outputdir
+from autogluon.core.utils.miscs import set_logger_verbosity
+import logging
+
+__all__ = ['Forecasting']
+
+logger = logging.getLogger()
+
+
+class Forecasting(BaseTask):
```

Comment:
This file seems very similar to predictor.py? Some of steps are the same. Maybe consider move those duplicated parts to utils

Reply:
This is because all those in the forecasting.py has been refactored into the predictor.py, just like what has been done in the tabular models. As the original task files have not been deleted in tabular, I did not delete this file in case there are some future usages..